src/content/advanced/iam-roles-for-service-accounts/index.md

```diff
@@ -0,0 +1,105 @@
+---
+linkTitle: IAM Roles for Service Accounts
+title: AWS IAM Roles for Service Accounts 
+description: This article describes how to use a new feature that allows binding of specific AWS IAM roles to a service account of a pod.
+weight: 60
+menu:
+  main:
+    parent: advanced
+user_questions:
+ -  How can I use IAM Roles for Service Accounts?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2022-03-04
+---
+
+# IAM Roles for Service Accounts (IRSA)
+
```

Comment:
The main headline is now added automatically, so it must not appear in the page body any more.

```suggestion

```